src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -240,8 +293,42 @@ A background pattern with the Merck cells as additional visual element.
+  </svg>
+</div>
+
+{% endexample %}
+
+## Animated
+
+Adds subtle animations to the background cell, to introduce some dynamics to the brand. Please use animations with caution to avoid user distraction.  
```

Comment:
```suggestion
Adds subtle animations to the background cells, to introduce some dynamics to the brand. Please use animations with caution to avoid user distraction.  
```